src/sls_web.js

```diff
@@ -6,12 +6,68 @@
 /* exported onOpen */
 /* exported parseFieldsAndCreateSlide */
 
+/**
+ * A special function that runs when the spreadsheet is open, used to add a
+ * custom menu to the spreadsheet.
+ */
+function onOpen() {
+  try {
+    loadConfiguration();
+    const spreadsheet = SpreadsheetApp.getActive();
+    const menuItems = [
+      {
+        name: 'Generate starter slide deck',
+        functionName: 'createStarterSlides',
+      },
+      {
+        name: 'Load configuration',
+        functionName: 'loadConfiguration',
+      },
+      {
+        name: 'Filter criteria only',
+        functionName: 'filterAndSortData',
+      },
+    ];
+    spreadsheet.addMenu('Performance Starter', menuItems);
+  } catch (error) {
+    throw new Error('onOpen failed: ' + error.toString());
+  }
+}
+
+/* eslint-disable no-unused-vars*/
```

Comment:
You won't need the no-unused-vars if you export them as above AFAIK

Reply:
Thank you!